Automattic/jetpack
# AI Assistant: Add translate action in extension

Close #31329 

## Proposed changes:

* Add text option in `i18nDropdrownControl`.
* Add `I18nDropdrownControl` in extension.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Enable `beta` extensions.
* Add `paragraph` block.
* Check if `Translate` is included in `Jetpack AI` toolbar option.

### Demo

![CleanShot 2023-06-14 at 11 46 34](https://github.com/Automattic/jetpack/assets/1663717/5ac4ad25-e187-4c9a-838e-94199dca7064)
